CONTRIBUTING.md

```diff
@@ -31,106 +41,112 @@ To send us a pull request, please:
+
+    - __Stable__
+```md
+<!--BEGIN STABILITY BANNER-->
+---
+
+![Stability: Stable](https://img.shields.io/badge/stability-Stable-success.svg?style=for-the-badge)
+
+> **This is a stable example. It should successfully build out of the box**
+>
+> This examples does is built on Construct Libraries marked "Stable" and does not have any infrastructure prerequisites to build.
+---
+<!--END STABILITY BANNER-->
 ```
-{
-  "name": [name of your application],
-  "version": "1.0.0",
-  "description": [description of your application],
-  "private": true,
-  "scripts": {
-    "build": "tsc",
-    "watch": "tsc -w",
-    "cdk": "cdk"
-  },
-  "author": {
-    "name": "Amazon Web Services",
-    "url": "https://aws.amazon.com",
-    "organization": true
-  },
-  "license": "Apache-2.0",
-  "devDependencies": {
-    "@types/node": "^8.10.38",
-    "typescript": "^3.2.4"
-  },
-  "dependencies": {
-    ...
-    "@aws-cdk/core": "^1.0.0"
-  }
-}
+
+- __Dev Preview__
+```md
+<!--BEGIN STABILITY BANNER-->
+---
+
+![Stability: Developer Preview](https://img.shields.io/badge/stability-Developer--Preview-important.svg?style=for-the-badge)
+
+> **This is an experimental example. It may not build out of the box**
+>
+> This examples does is built on Construct Libraries marked "Developer Preview" and may not be updated for latest breaking changes.
+>
+> It may additionally requires infrastructure prerequisites that must be created before successful build.
+>
+> If build is unsuccessful, please create an [issue](https://github.com/aws-samples/aws-cdk-examples/issues/new) so that we may debug the problem 
+---
+<!--END STABILITY BANNER-->
 ```
 
-## Style and Formatting
+  - Overview: This should be a description of the example including, but not limited to, an overview of resources being created in the example, a description of any standard/common patterns being used, as well as the overall function of the example.
+  - Structure: If the example is more complex than a single stack, or the stack contains more than a couple resources, please describe the structure of application being deployed. This can be as simple as a bulleted list of resources, or as intricate as a complete infrastructure diagram.
+  - Build/Deploy: Finally, the README must include the basic build/deploy instructions required for that example in said language.
+
+
+## Style and Formatting <a name="Style"></a>
 
 We strive to keep examples consistent in style and formatting. This hopefully makes navigating examples easier for users. Since the examples span different languages, we try to keep example code as idiomatic as possible.
 
 New guidelines for various languages will be added as we define them.
 
+### Typescript
+1. You can use [eslint](https://eslint.org/) with the standard configuration to check for code syntax.
```

Comment:
**check for code syntax** -> **check code syntax** or **check syntax and style** (since a linter does both)

Reply:
```suggestion
1. You can use [eslint](https://eslint.org/) with the standard configuration to check code syntax and style.
```